docs/changehistory/NextVersion.md

```diff
@@ -206,3 +203,32 @@ Some have replacements within the @bentley/ui-core package.
 | DialogButtonType in @bentley/ui-core  | DialogButtonType in @bentley/ui-abstract   |
 | LocalUiSettings in @bentley/ui-core   | LocalSettingsStorage in @bentley/ui-core   |
 | SessionUiSettings in @bentley/ui-core | SessionSettingsStorage in @bentley/ui-core |
+
+### New @bentley/ui-imodel-components package
+
+A new @bentley/ui-imodel-components package has been added, and some items were moved from @bentley/ui-core and @bentley/ui-components into this new package.
+The ui-imodel-components package contains React components that depend on the imodeljs-frontend, imodeljs-common or imodeljs-quantity packages.
+Dependencies on these other iTwin.js packages have been removed from ui-core and ui-components.
+The items moved to ui-imodel-components are related to Color, Cube, LineWeight, Navigation Aids, Quantity Inputs, Timeline and Viewport.
+
+The following items were moved into the ui-imodel-components package. For a complete list, see [iTwin.js Documentation](https://www.itwinjs.org/reference/ui-imodel-components/all).
```

Comment:
Can you make this a relative link?

Reply:
I don't know what the relative link would be. I've seen in 2.18.0.md where it uses the full path when referring to a package such as @bentley/ecschema-metadata. Note that I'm attempting to link to the full index for @bentley/ui-imodel-components.

Reply:
@williamkbentley is that possible to do? I thought there was a syntax for this.

Reply:
In the UI packages, while we have removed several old deprecated APIs and components, 3.0 will still contain some old deprecated APIs dealing with UI 2.0, which apps have been slow to adopt, and newly deprecated APIs when an iTwinUI-react component is preferred.

Reply:
Looks like Will is out. Can Paul or Caleb resolve this conversation and approve?

Reply:
@DanEastBentley I'll resolve and will look for the docs syntax I wanted to use. We can follow up in a separate PR.